MobileWallet/Common/Theme.swift

```diff
@@ -377,6 +277,118 @@ struct Images: Loopable {
     let settings = UIImage(named: "settings")
 }
 
+struct Fonts: Loopable {
+    let splashVersionFooterLabel = UIFont(avenirStyle: UIFont.AvenirStyle.heavy, size: 9.0)
+    let actionButton = UIFont(avenirStyle: UIFont.AvenirStyle.heavy, size: 16.0)
+    let copiedLabel = UIFont(avenirStyle: UIFont.AvenirStyle.black, size: 13.0)
+
+    //Splash
+    let splashTitleLabel = UIFont(avenirStyle: UIFont.AvenirStyle.black, size: 30.0)
```

Comment:
Can't you have something like `let splashTitleLabel = Avenir.black.withSize(30.0)`?

Reply:
Or `Font.Avenir.black.withSize(30.0)`, or `UIFont.Avenir.black.withSize(30.0)`.